pkg/test/resourcefixture/contexts/accesscontextmanager_context.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package contexts
+
+func init() {
+
+	resourceContextMap["accesscontextmanagerserviceperimeterresource"] = ResourceContext{
+		ResourceKind: "AccessContextManagerServicePerimeterResource",
+		SkipUpdate:   true,
```

Comment:
Could you add the comment with the context about why update is skipped? https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.NewResourceFromTerraform.md#optionally-add-the-resource-to-test-contexts

Reply:
`update` is not mentioned in the TF doc: https://registry.terraform.io/providers/hashicorp/google/4.84.0/docs/resources/access_context_manager_service_perimeter_resource#timeouts, so I think we should skip update. Added comment.

Reply:
Thank you! Have you verified whether it's the API doesn't support update or TF doesn't support update?
I probably will not mention TF unless it's the TF blocks KCC from testing update (i.e. if the API supports update, but TF doesn't, then saying `TF doesn't support update` makes more sense). If it is not supported in the API nor TF, `Update is not supported for this resource` seems to be a better comment.

This is not a blocker.